### PR TITLE
Create UI to update user email

### LINKED ISSUE #348: Create an user interface to update the user email
**Is your feature request related to a problem? Please describe.**
- As a user, I should be able to update my email and confirm the email is correct.

**Describe the solution you'd like**
- Redirect the user to the email update view in the initial login. 
- User can change the email or confirm the email that we get from LinkedIn (Use `hasUpdateUserDetails` to check this)
- Even after the initial login this UI can be accessed

### PR DESCRIPTION
**Pull request by YaiyaSam**
## Purpose

The purpose of this PR is to fix #348

## Goals

To create a new ui to update user email

## Approach

Added a new `Settings` page to the navbar of the home page

## Screenshots

image

### CODE REVIEW

**Comment by anjula-sack** [comment]
Good job! @YaiyaSam Can we have different content at the initial login? Like, Confirm Email and just an input? After that, It will be like yours.

**Comment by YaiyaSam** [comment]
> Good job! @YaiyaSam Can we have different content at the initial login? Like, Confirm Email and just an input? After that, It will be like yours.

sorry just saw this. what I was going to do was...have something similar to the loginmodal to appear on the home page asking if the current email is the one they are using. and if the user select yes, that would just the update the db, if the user selects no, then they would get redirected to this page to update the email. 
which one should I go with?

**Comment by anjula-sack** [comment]
> > Good job! @YaiyaSam Can we have different content at the initial login? Like, Confirm Email and just an input? After that, It will be like yours.
> 
> sorry just saw this. what I was going to do was...have something similar to the loginmodal to appear on the home page asking if the current email is the one they are using. and if the user select yes, that would just the update the db, if the user selects no, then they would get redirected to this page to update the email. which one should I go with?

I think modal is good cuz we don't have much content to show as well. Your solution is better

File: src/scenes/Home/scenes/Settings/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+.email_edit_title {
+    margin-left: 140px;
+}
+.backButton {
+    margin-top: 1rem;
+    border: transparent;
+    background: transparent;
+    box-shadow: none;
+    margin-left: 70px;
+}
```
Comment (anjula-sack) [inline src/scenes/Home/scenes/Settings/styles.css]: EOL

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/scenes/Home/scenes/Settings/styles.css (hunk 1 above)
Comment (anjula-sack) [reply src/scenes/Home/scenes/Settings/styles.css]: can you add the extra line here? @YaiyaSam

Comment (YaiyaSam) [reply src/scenes/Home/scenes/Settings/styles.css]: yeah ill add tht

File: src/components/EmailModal/interfaces.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface EmailModalProps {
+  isModalVisible: boolean;
+  onCancel: () => void | null;
```
Comment (anjula-sack) [inline src/components/EmailModal/interfaces.ts]: why do we need the `null` type here?

File: src/scenes/Home/scenes/Settings/styles.css (hunk 1 above)
Comment (YaiyaSam) [reply src/scenes/Home/scenes/Settings/styles.css]: @anjula-sack is this fixed now?

File: src/components/EmailModal/interfaces.ts (hunk 2 above)
Comment (YaiyaSam) [reply src/components/EmailModal/interfaces.ts]: oh yeah i can get rid of the null for both onCancel and onConfirm ryt

Comment (YaiyaSam) [reply src/components/EmailModal/interfaces.ts]: i removed it. pls review

**Review by anjula-sack — APPROVED** [review]
Thanks for getting this done! @YaiyaSam

**Pull request merged**